### ISSUE TITLE
Get started with Android SDK

### ISSUE THREAD

**Issue by dondi** [OceanHelloWorld/AR-for-sports#2]
Initial exploration: download the Android SDK and get yourselves set up with it. Get a Hello World and additional "conventional" apps up and running. Target getting some familiarity by Saturday May 21.

Some thoughts on getting started:
- Get comfortable with the development environment.
- Be aware of the files involved. Differentiate between the ones you maintain vs. the ones built/created by the SDK.
- Write a few simple apps to get a feel for the API. Android versions of homework assignments are a good pool to try out, because at least you have written them before.
- You can look at the specific APIs we'll need (camera, graphics) if you like, but let yourself learn the fundamentals first.
- No need to commit this code (well, to this repo at least); these are all preliminary.
- Per the milestone, give yourselves until Saturday May 21 to get comfortable. Of course you can take more time, but we have a tight schedule here.

**Comment by dondi** [comment]
Status report? Can you guys build a basic Android app from scratch now?

**Comment by jkkealii** [comment]
Yes! I've been working through a tutorial on getting familiar with android development and I would say I am 50% comfortable with building a functional app, though I still need to cover many fundamental topics.
